src/Symfony/Upgrade/Fixer.php

```diff
@@ -29,13 +30,30 @@ public function __construct(
         $this->stopwatch = $stopwatch;
     }
 
-    public function registerBuiltInFixers()
+    public function registerAllBuiltInFixers()
     {
+        /** @var AbstractFixer $fixer */
         foreach (new FixerIterator() as $fixer) {
             $this->addFixer($fixer);
         }
     }
 
+    /**
+     * @param array $fixerNameList
+     */
+    public function registerBuiltInFixers(array $fixerNameList)
+    {
+        /** @var AbstractFixer $fixer */
+        foreach (new FixerIterator() as $fixer) {
+            if (in_array($fixer->getName(), $fixerNameList, true)) {
+                $this->addFixer($fixer);
+            }
+        }
+    }
+
+    /**
+     * @param FixerInterface $fixer
```

Comment:
This brings no value, it's already in the code, please remove.

Reply:
done